lms/views/lti/oidc.py

```diff
@@ -59,8 +59,12 @@ def oidc_view(request):
 
     registration = request.find_service(LTIRegistrationService).get(issuer, client_id)
     if not registration:
-        raise HTTPForbidden(
-            f"""Registration not found. iss:'{issuer}', client_id:'{client_id}'"""
+        raise ValidationError(
```

Comment:
Raise ValidationError for missing registration instead of HTTPForbidden 

ValidationError gives a better error as it not as easily confused
with something related to the current user (ie missing roles) as HTTPForbidden

This is an error that would happen in every launch so the
first user to see it will likely by the LMS admin.

We include the word "LTI" to better direct the user in the right
direction.